### DISCUSSION TITLE
Connecting Azure Databricks Tables and Analyzing Extracted Data - By Autogen agents

### DISCUSSION THREAD

**Discussion by mraguth** [microsoft/autogen#1384] [General]
Is it possible to connect Azure Databricks tables using any of the autogen agents and retrieve data from Databricks? The goal is to analyze the extracted data from Databricks via autogen agents.

**Comment by ekzhu** [comment]
I think you can build this feature with function call. If you can define a list of operations and write function to implement those operations. You can register those functions to the agents, and they will be able to use them.

**Reply by mraguth** [reply]
Can you provide some example code, please? especially in connecting external sources.
